### PR TITLE
Add BunkerWeb in WAFs

### PR DESCRIPTION
**Pull request by LeoSpyke**
Add BunkerWeb in Web Application Firewall section, removed the old reference with a wrong naming.

Docs: https://docs.bunkerweb.io/latest/

### CODE REVIEW

**Review by SofianeHamlaoui — APPROVED** [review]

**Comment by LeoSpyke** [comment]
waiting for @sbilly approval

**Review by andrzejsydor — APPROVED** [review]

**Pull request merged**
